public/fischLogo.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const FischLogo = () => {
+  return (
+    <svg
```

Comment:
**issue:** React components should not be placed in `public` directory. Keep it in `components` like any other component. If you want to keep SVG asset in `public` you should keep it in `.svg` file extension